hac-node/build/make20.bash

```
@@ -0,0 +1,89 @@
+
+declare -a NODE_IDS
+for i in $(seq 1 $((NUM_NODES))); do
+    NODE_IDS[$i]=$(cometbft show-node-id --home data$i)
+    echo ${NODE_IDS[$i]}
+done
+
+for i in $(seq 1 $((NUM_NODES))); do
+    sleep 1
+    P2P_PORT=$((BASE_P2P_PORT + (i-1)*PORT_INTERVAL))
+    RPC_PORT=$((BASE_RPC_PORT + (i-1)*PORT_INTERVAL))
+    APP_PORT=$((BASE_APP_PORT + (i-1)*PORT_INTERVAL))
```

Shellcheck (0.10.0)

[warning] APP_PORT appears unused. Verify use (or export if used externally).

(SC2034)